package.json

```diff
@@ -34,6 +35,7 @@
     "history": "1.12.6",
     "immutable": "^3.7.5",
     "isomorphic-fetch": "^2.2.0",
+    "npm-scripts-info": "^0.1.0",
```

Comment:
Oh, I just realized this is in `dependencies`. What do you think about moving it to `devDependencies`?